Demos/Parallax/Parallax/AppDelegate.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+  lazy var navigationController: UINavigationController = { [unowned self] in
```

Comment:
Somone said that all variables were lazy in Swift 1.2, so the `lazy` might not be needed